packages/build/src/commands/core_command.js

```diff
@@ -16,12 +18,21 @@ const fireCoreCommand = async function ({
   nodePath,
   childEnv,
   envChanges,
+  errorParams,
+  configOpts,
   netlifyConfig,
+  priorityConfig,
   featureFlags,
+  debug,
 }) {
   try {
+    const configSideFiles = await listConfigSideFiles(netlifyConfig, buildDir)
     const childEnvA = setEnvChanges(envChanges, { ...childEnv })
-    return await coreCommand({
+    const {
+      newEnvChanges = {},
+      configMutations = [],
```

Comment:
Bonus from this PR: core commands can now change environment variables and `netlifyConfig` by returning a `newEnvChanges` and `configMutations` variables.